### ISSUE TITLE
Opening Project Error Message

### ISSUE THREAD

**Issue by EKFREIBURGER** [olivierfriard/BORIS#731]
Version: v.8.24.1 2024-02-13

**Describe the bug**
Project was edited and saved- and exported to desktop. Project was attempted to be re-opened from recent export to desktop. Upon re-opening project, error message appears. See Image below of error message. Project will not open due to error message. Content from project cannot be recovered at this time. 

![Screenshot (6)] 
(https://github.com/olivierfriard/BORIS/assets/164543453/c9e0512b-d099-465d-9380-8a3afe312130)

**To Reproduce**
1. Open BORIS software. 
2. Select Project 
3. Select Open Project 
4. Select exported project file 
5. Error Message appears on screen

**Expected behavior**
I expected the project to re-open like any other project does upon opening it from the boris.file download

**Screenshots**
See attached image above.

**Desktop (please complete the following information):**
 - OS: [windows 10 enterprise, v.21H2 OS Build: 19044.3208]
 - Type of BORIS instalaltion - is portable

**In case of crash** attach or copy the content of the boris.log file  C:\Users\ekfreiburger\boris.log

**Comment by AdamGLIN** [comment]
Hello !

I think the problem might come from an unexpected invisible character in your file name, maybe renaming it (for example without spaces) would be a good idea ?

I think [this](https://stackoverflow.com/questions/42019117/unicodedecodeerror-charmap-codec-cant-decode-byte-0x8f-in-position-xxx-char) refers this issue and explain how to fixe it.

Hope it helps !

**Comment by olivierfriard** [comment]
I do not think that the problem is due to the file name but to the content of the file.
The error indicates that a character can not be decoded at position 8833.
If it is possible send me your project file by email, I will check it.
